traffic_ops/app/lib/API/Tenant.pm

```diff
@@ -44,14 +44,16 @@ sub index {
 	my $orderby = $self->param('orderby') || "name";
 	my $rs_data = $self->db->resultset("Tenant")->search( undef, {order_by => 'me.' . $orderby } );
```

Comment:
here's an example where a query is limited by "in"

https://github.com/apache/incubator-trafficcontrol/blob/master/traffic_ops/app/lib/API/Cdn.pm#L701